### ISSUE TITLE
Make VariableDeclarator understand named imports

### ISSUE THREAD

**Issue by rotsee** [eslint/eslint#10490]
**What rule do you want to change?**
`indent`, specifically `VariableDeclarator` behaviour

**Does this change cause the rule to produce more or fewer warnings?**
fewer

**How will the change be implemented? (New option, new default behavior, etc.)?**
It seems to me this should be the default behavior, for consistency.

**Please provide some example code that this change will affect:**

```js
const {foo, bar, fiz, 
      buz, foobar} = require("xyzzy")
```

Using the following rule (an example from the [docs](https://eslint.org/docs/rules/indent#variabledeclarator)):
```js
2, { "VariableDeclarator": { "var": 2, "let": 2, "const": 3 } }
```

**What does the rule currently do for this code?**
`error  Expected indentation of 2 spaces but found 6  indent`

**What will the rule do after it's changed?**
No error, just as the code below produces no error:

```js
const foo, bar, fiz,
      buz, foobar = require("xyzzy")
```

**Comment by rotsee** [comment]
I see the issues this would cause. Thank you for the alternative suggestion!
